### ISSUE TITLE
8.2.0.0 patch is broken

### ISSUE THREAD

**Issue by EugeneTkachenko** [SitecoreSupport/Sitecore.Support.130279#2]
Patch at  
https://github.com/SitecoreSupport/Sitecore.Support.130279/releases/tag/8.2.0.0
resource is broken and has not all files inside (no more deep checking was performed).

**Comment by AlenPelin** [comment]
Agree, it misses necessary files and breaks [versioning policy](https://github.com/SitecoreSupport/Readme/blob/master/VERSIONING.md) (must be `1.3.0`, not `8.2.0`)

**Comment by AlenPelin** [comment]
It is a little bit embarrassing that AY (he doesn't work here any more) tested non-working patch and it fixed the issue :)

> Edited by Alexander Yaremenko on Monday, May 29, 2017 at 1:18 PM (UTC)
> 
> Hi Rob,
> 
> I have installed this patch on my Sitecore instance and the issue is gone.
